src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -10,6 +10,7 @@
 underlying data-services.
 """
 
+import json
```

Comment:
Valentin, this import won't be used 
Since there is following on line 18
from WMCore.Wrappers import JsonWrapper as json
https://github.com/vkuznet/WMCore/blob/pycurl-fix/src/python/WMCore/Services/pycurl_manager.py#L18